pkg/streamingpromql/engine_test.go

```diff
@@ -48,12 +48,11 @@ func TestUnsupportedPromQLFeatures(t *testing.T) {
 		"metric{} or other_metric{}": "binary expression with many-to-many matching",
 		"metric{} + on() group_left() other_metric{}":  "binary expression with many-to-one matching",
 		"metric{} + on() group_right() other_metric{}": "binary expression with one-to-many matching",
-		"avg(metric{})":                         "aggregation operation with 'avg'",
```

Comment:
Perhaps we can replace this with a different aggregation operation we don't yet support, like `count` or `group`?